src/plcc.py

```diff
@@ -96,16 +96,6 @@ def main():
             argv = argv[1:]
         else:
             break
-    
-    # Handle --version option.
-    if 'version' in flags and flags['version']:
-        from pathlib import Path
-        version_file = Path(__file__).resolve().parent / 'VERSION'
-        with open(version_file, 'r') as f:
-            contents = f.read()
-            print("PLCC " + contents.strip())
-        sys.exit(0)
-
```

Comment:
These lines do not have anything to do with the changes being made. Removing them breaks the `plcc --version` feature. Please return them.